Index: tests/test_second_variant.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from selenium.webdriver import Keys, ActionChains\r\nfrom selenium.webdriver.common.by import By\r\n\r\n\r\n# Second way. LoudPass. Do one napas. And is not санина, ебаная. NOT\r\n\r\n\r\ndef test_eto_pizda_test(setup):\r\n    driver = setup\r\n    general_window = driver.current_window_handle\r\n    headers_off_all_articles = driver.find_elements(By.XPATH, \"//div[@class = 'title']\")\r\n\r\n    for header_of_one_article in headers_off_all_articles:\r\n        text_of_one_header = header_of_one_article.text\r\n        ActionChains(driver).key_down(Keys.LEFT_CONTROL).click(header_of_one_article) \\\r\n            .key_up(Keys.LEFT_CONTROL).perform()\r\n        driver.switch_to.window(driver.window_handles[1])\r\n        assert driver.find_element(By.XPATH, \"//h1/i\").text == text_of_one_header\r\n        driver.close()\r\n        driver.switch_to.window(general_window)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/tests/test_second_variant.py b/tests/test_second_variant.py
--- a/tests/test_second_variant.py	(revision 7ccfc952bbfc56c95ea822ee92a80bd8be07255e)
+++ b/tests/test_second_variant.py	(date 1675001845075)
@@ -2,7 +2,7 @@
 from selenium.webdriver.common.by import By
 
 
-# Second way. LoudPass. Do one napas. And is not санина, ебаная. NOT
+# Second way. LoudPass.
 
 
 def test_eto_pizda_test(setup):
