Index: tests/test__equals_headers.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/tests/test__equals_headers.py b/tests/test__equals_headers.py
--- a/tests/test__equals_headers.py	
+++ b/tests/test__equals_headers.py	
@@ -1,6 +1,6 @@
 from selenium.webdriver import Keys, ActionChains
 from selenium.webdriver.common.by import By
-# First way
+
 
 def test_eto_pizda_test(setup):
     driver = setup
