Index: tests/on_a_future.py
===================================================================
diff --git a/tests/on_a_future.py b/tests/on_a_future.py
--- a/tests/on_a_future.py	
+++ b/tests/on_a_future.py	
@@ -1,9 +1,9 @@
-# from locators.locators import Locators
-# from pages.form_page import FormPage
-#
-# #  Допилим.
-# class TestPizdaTest:
-#
-#     def test_eto_pizda_test(self, start_driver):
-#         all_thems = FormPage.all_elements_what_we_are_need
-#         print(all_thems)
\ No newline at end of file
+from locators.locators import Locators
+from pages.form_page import FormPage
+
+#  Допилим.
+class TestPizdaTest:
+
+    def test_eto_pizda_test(self, start_driver):
+        all_thems = FormPage.all_elements_what_we_are_need
+        print(all_thems)
\ No newline at end of file
Index: tests/test_second_variant.py
===================================================================
diff --git a/tests/test_second_variant.py b/tests/test_second_variant.py
--- a/tests/test_second_variant.py	
+++ b/tests/test_second_variant.py	
@@ -5,14 +5,18 @@
 
 def test_eto_pizda_test(setup):
     driver = setup
-    general_window, all_themes = driver.current_window_handle, driver.find_elements(By.XPATH, "//div[@class = 'title']")
-    themes_list = [themes_list for themes_list in all_themes]
+    general_window = driver.current_window_handle
+    headers_of_every_articles = driver.find_elements(By.XPATH, "//div[@class = 'title']")
 
-    for one_theme in range(len(themes_list)):
-        ActionChains(driver).key_down(Keys.LEFT_CONTROL).click(themes_list[one_theme]).perform()
+    for header_of_one_article in headers_of_every_articles:
+        text_of_one_header = header_of_one_article.text
+        ActionChains(driver).key_down(Keys.LEFT_CONTROL).click(header_of_one_article)\
+            .key_up(Keys.LEFT_CONTROL).perform()
+
         driver.switch_to.window(driver.window_handles[1])
-        text_for_assert = driver.find_element(By.XPATH, "//h1/i").text
+
+        assert text_of_one_header == driver.find_element(By.XPATH, "//h1/i").text
+
         driver.close()
+
         driver.switch_to.window(general_window)
-
-        assert text_for_assert == themes_list[one_theme].text
